docs/quickstart.rst

```diff
@@ -197,18 +202,39 @@ function.  Optionally a converter can be used by specifying a rule with
         # show the post with the given id, the id is an integer
         return 'Post %d' % post_id
 
-The following converters exist:
+Flask builds on the routing system provided by Werkzeug which provides
+the following converters to be used in your application's routes:
+
+========================================================  ===============================================
+`string` (:class:`~werkzeug.routing.UnicodeConverter`)    accepts any text without a slash (the default)
+`int` (:class:`~werkzeug.routing.IntegerConverter`)       accepts integers
+`float` (:class:`~werkzeug.routing.FloatConverter`)       like `int` but for floating point values
+`path` (:class:`~werkzeug.routing.PathConverter`)         like the default but also accepts slashes
+`any` (:class:`~werkzeug.routing.AnyConverter`)           matches one of the items provided
+`uuid` (:class:`~werkzeug.routing.UUIDConverter`)         accepts UUID strings
+========================================================  ===============================================
+
+Building custom converters is also possible by subclassing
+the :class:`~werkzeug.routing.BaseConverter` then registering your
+custom converter against :meth:`flask.Flask.url_map.converters`::
+
+    from werkzeug.routing import BaseConverter
```

Comment:
I merged a PR a while ago that expanded this list in the first place, should have caugth that.  It needs to be *reduced* to an actual quickstart here and link to the api docs:

> The default converter is `string`.  Other converters are available as well, see [URL route registrations](http://flask.pocoo.org/docs/0.11/api/#url-route-registrations) to learn more.  Here are some examples:

The api docs should include all this new information.